tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -112,12 +120,60 @@ def validate_dut_routes_exist(duthosts, rand_one_dut_hostname, dut_dhcp_relay_da
         assert len(rtInfo["nexthops"]) > 0, "Failed to find route to DHCP server '{0}'".format(dhcp_server)
 
 
-def test_dhcp_relay_default(duthosts, rand_one_dut_hostname, ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist):
+def restart_dhcp_service(duthost):
+    duthost.shell('systemctl restart dhcp_relay')
+    duthost.shell('systemctl reset-failed dhcp_relay')
+
+    for retry in range(5):
+        time.sleep(30)
+        dhcp_status = duthost.shell('docker container top dhcp_relay | grep dhcrelay | cat')["stdout"]
```

Comment:
can you please add an extra checkout to make sure the dhcrelay params match the mode?

Reply:
i believe it is not necessary

Reply:
You'd capture an error instead of trying to figure out which mode dhcp is. This is test and we need to make sure it is running in dualtor mode when the config db is set. It is as if we test the template is working as well

Reply:
if something is wrong in parsing db and running process, packet level tests will definitely fail. The only purpose here to ensure that we pause testing until processes start to run.